fw/http_parser.c

```diff
@@ -2379,7 +2410,7 @@ __req_parse_cache_control(TfwHttpReq *req, unsigned char *data, size_t len)
 		/* Any directive we don't understand.
 		 * Here we just skip all the tokens, double quotes and equal signs.
 		 */
-		__FSM_I_MATCH_MOVE_fixup(qetoken, Req_I_CC_Ext, 0);
+		__FSM_I_MATCH_MOVE(qetoken, Req_I_CC_Ext);
```

Comment:
OK, I ran the test and see the failure on checking the Cache-Control parsing results. I ran the test with `DBG_HTTP_PARSER=3` and for these chunks: `: max-age=1, dum` and `my, no-store, mi` I see following messages:

```
parser at Req_I_CC: c=0x64(d), p_off=11
parser at Req_I_CC_Ext: c=0x64(d), p_off=11
store field chunk len=3 data=0000000038ad924b(d) field=<0x0,15,00000000bea6751e>
```
This looks reasonable that we fixup the `dum` part. For the next chunk I see
```
parser at Req_I_CC_Ext: c=0x6d(m), p_off=0
parser at Req_I_EoT: c=0x2c(,), p_off=2
```
I can not say anything specific for this results. Need to check what we actually have in `h_cc`. So the fix isn't finally clear. It's also not clear whether we have the same issue in other places since the root of the problem isn't described neither in the commit, nor in the original issue.

Also my previous comment about double state processing still must be addressed.